config/software/chef.rb

```diff
@@ -74,6 +74,11 @@
   # Don't use -n #{install_dir}/bin. Appbundler will take care of them later
   gem "install chef*.gem --no-ri --no-rdoc --verbose", env: env
 
+  # ensure we put the gems in the right place to get picked up by the publish scripts
```

Comment:
Might be worth adding a bit more context so it's easier for a future person to know when this can be changed/removed.